api/package.json

```diff
@@ -15,26 +15,25 @@
         "co-postgres-queries": "~0.2.2",
         "convict": "^5.2.0",
         "express": "^4.17.1",
-        "object-hash": "~1.3.1",
-        "slug": "~1.1.0",
-        "uuid": "^3.3.3"
```

Comment:
`uuid`  is no longer in use?

Reply:
It was never used :/
I don't remember why it was installed

Reply:
^^ Ok